packer/sagebio/sagebio-base-ubuntu-bionic-LATEST/playbook.yaml

```diff
@@ -18,10 +18,11 @@
         state: present
 
     # awscli is on aws linux2 distro by default
-    - name: install awscli
+    - name: install awscli and SSM
       snap:
         name:
           - aws-cli
+          - amazon-ssm-agent
```

Comment:
what about linux2 distro?  is the ssm agent installed on that by default?

Reply:
Yes, it is installed on the base image of linux2